src/index.test.js

```diff
@@ -85,6 +85,19 @@ test('options: logLevel sets the log level', () => {
   })
 })
 
+test('options: scripts prevents command text when false', () => {
+  const {runPackageScript} = setup()
+  const scriptConfig = {test: {script: 'echo test'}}
+  const options = {scripts: false}
+  return runPackageScript({
+    scriptConfig,
+    scripts: ['test'],
+    options,
+  }).then(() => {
+    // expect(stdout).toEqual(...)
```

Comment:
@kentcdodds Could you advise on how you would capture stdout to test this? Currently no precedent in tests (that I've seen).